server/src/core/auth/dto/save-connected-account.ts

```diff
@@ -0,0 +1,31 @@
+import { ArgsType, Field } from '@nestjs/graphql';
+
+import { IsNotEmpty, IsString } from 'class-validator';
+
+@ArgsType()
+export class SaveConnectedAccountInput {
```

Comment:
I'm surprised about this class, connectedAccount is a standard object, it should not be in src/core